index/manual_plantsvszombies2gardendless_trikehard.toml

```diff
@@ -0,0 +1,8 @@
+name = "manual_plantsvszombies2gardendless_trikehard"
```

Comment:
This needs to match the game name in the YAML (`Manual_gamename_author`) and is case sensitive